assessment/collectors/parser/response_parser.go

```diff
@@ -72,38 +78,33 @@ func parseAnyToInteger(anyType any) int {
 	return i
 }
 
-func ParseSchemaImpact(schemaImpactResponse map[string]any, projectPath, filePath string) (*Snippet, error) {
-	logger.Log.Debug("schemaImpactResponse:", zap.Any("sec: ", schemaImpactResponse))
-	return &Snippet{
-		SchemaChange:          parseAnyToString(schemaImpactResponse["schema_change"]),
-		TableName:             parseAnyToString(schemaImpactResponse["table"]),
-		ColumnName:            parseAnyToString(schemaImpactResponse["column"]),
-		NumberOfAffectedLines: parseAnyToInteger(schemaImpactResponse["number_of_affected_lines"]),
-		SourceCodeSnippet:     ParseStringArrayInterface(schemaImpactResponse["existing_code_lines"]),
-		SuggestedCodeSnippet:  ParseStringArrayInterface(schemaImpactResponse["new_code_lines"]),
-		RelativeFilePath:      getRelativeFilePath(projectPath, filePath),
-		FilePath:              filePath,
-		IsDao:                 true,
-	}, nil
+func ParseAnyToBool(anyType any) bool {
+	str := ParseAnyToString(anyType)
+	i, err := strconv.ParseBool(str)
+	if err != nil {
+		logger.Log.Debug("could not parse string to int" + str)
+		return false
+	}
+	return i
 }
 
 func ParseCodeImpact(codeImpactResponse map[string]any, projectPath, filePath string) (*Snippet, error) {
 	//To check if it is mandatory for the response to contain these methods
 	return &Snippet{
-		SourceMethodSignature:    parseAnyToString(codeImpactResponse["original_method_signature"]),
-		SuggestedMethodSignature: parseAnyToString(codeImpactResponse["new_method_signature"]),
+		SourceMethodSignature:    ParseAnyToString(codeImpactResponse["original_method_signature"]),
```

Comment:
not necessarily in this PR, but we should be using constants here

Reply:
notes, will make the change in a follow up PR